Bug Issue 2

Assets\scripts\GameManager.cs(68,3): error CS0103: The name '_score' does not exist in the current context

Assets\scripts\GameManager.cs(69,20): error CS0029: Cannot implicitly convert type 'int' to 'string'

--- a/Assets/scripts/GameManager.cs
+++ b/Assets/scripts/GameManager.cs
@@ -65,8 +65,8 @@ public class GameManager : MonoBehaviour {
 
 	void OnPlayerScored(){
 	
-		_score++;
-		scoreText.text = _score;
+		score++;
+		scoreText.text = score.ToString();
 	}
 
 	void SetPageState(PageState state){

